aicsimageio/aics_image.py

```diff
@@ -106,6 +55,45 @@ def __init__(
         kwargs: Dict[str, Any]
             Extra keyword arguments that can be passed down to either the reader subclass or, if using the context
             manager, the LocalCluster initialization.
+
+        Examples
+        --------
+        Initialize an image and read the slices specified as a numpy array.
+
+        >>> img = AICSImage("my_file.tiff")
+        ... zstack_t8 = img.get_image_data("ZYX", S=0, T=8, C=0)
+
+        Initialize an image, construct a delayed dask array for certain slices, then read the data.
+
+        >>> img = AICSImage("my_file.czi")
+        ... zstack_t8 = img.get_image_dask_data("ZYX", S=0, T=8, C=0)
+        ... zstack_t8_data = zstack_t8.compute()
+
+        Initialize an image and pass arguments to the reader using kwargs.
+
+        >>> img = AICSImage("my_file.czi", chunk_by_dims=["T", "Y", "X"])
+
+        Create a local dask cluster for the duration of the context manager.
+
+        >>> with AICSImage("filename.ome.tiff") as img:
+        ...     data = img.get_image_data("ZYX", S=0, T=0, C=0)
+
+        Create a local dask cluster with arguments for the duration of the context manager.
+
+        >>> with AICSImage("filename.ome.tiff", dask_kwargs={"nworkers": 4}) as img:
+        ...     data = img.get_image_data("ZYX", S=0, T=0, C=0)
+
+        Connect to a specific dask cluster for the duration of the context manager.
+
+        >>> with AICSImage("filename.ome.tiff", dask_kwargs={"address": "tcp://localhost:1234"}) as img:
+        ...     data = img.get_image_data("ZYX", S=0, T=0, C=0)
+        ```
+
+        Notes
```

Comment:
Can you still create a AICSImage by passing a ndarray or dask array into it?

Reply:
Yes just removed those examples. I can add them back if desired.